assets/app/view/game/convert.rb

```diff
@@ -0,0 +1,26 @@
+# frozen_string_literal: true
+
+require 'view/game/actionable'
+
+module View
```

Comment:
why do you need a new view for this? does this mess up 1817?

Reply:
The conversion happens at the start of a corporations operating turn instead of a separate round like 1817 so the conversion button needs to be hooked into the operating round view.

The new Convert view is nice because conversion is simply a yes/no choice, if it's a choice at all.

I haven't checked 1817 on this branch yet but I will later this evening

Reply:
![image](https://user-images.githubusercontent.com/2993555/105662136-f861cc00-5e9c-11eb-850c-62f4016f3d66.png)
1817 still works as it did